recipe/meta.yaml

```diff
@@ -17,11 +17,15 @@ package:
   version: {{ version }}
 
 source:
-  url: https://www.neuron.yale.edu/ftp/neuron/versions/v{{ xy }}/{{ version }}/nrn-{{ version }}.tar.gz
-  sha256: 85a0f999a9cdcc661b0066c32a78239e252f26e74eee5328bf1106393400bfc0
-  patches:
-    - 0001-memacs.patch
-    - 0002-Update-deprecated-MPI_Adress-and-MPI_Type_struct.-Fix-if-a=b-warning.patch
+  - url: https://www.neuron.yale.edu/ftp/neuron/versions/v{{ xy }}/{{ version }}/nrn-{{ version }}.tar.gz
+    sha256: 85a0f999a9cdcc661b0066c32a78239e252f26e74eee5328bf1106393400bfc0
+    patches:
+      - 0001-memacs.patch
+      - 0002-Update-deprecated-MPI_Adress-and-MPI_Type_struct.-Fix-if-a=b-warning.patch
+    folder: yale
+  - url: https://github.com/neuronsimulator/nrn/archive/{{ version }}.tar.gz
```

Comment:
The Python package builds are incomplete due to missing files in the https://www.neuron.yale.edu/ftp/neuron/versions/v7.7/ source archives.
xref: https://github.com/neuronsimulator/nrn/issues/411